stanford_person.post_update.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/**
+ * @file
+ * File description.
+ *
+ * Long description.
+ */
+
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\file\Entity\File;
+use Drupal\media\Entity\Media;
+
+/**
+ * Install the default profile image.
+ *
+ * Add the image to the media library and add the entities to the DB.
+ */
+function stanford_person_post_update_8100() {
```

Comment:
Since all of this is copied from the install hook, I think it should be a separate function that is called in the two hooks.

Reply:
Good idea. 

Overall, adding the default image vs altering the layout/template to fall back to the image. Which way do you prefer?

Reply:
i'd vote the default image in config. it'll be easier to track down if we need to change it in the future.